docs/wxm-devices/pulse-bundle/maintenance.mdx

```diff
@@ -0,0 +1,40 @@
+---
+id: maintenance
+title: Maintenance of Pulse bundle (WB3000)
+sidebar_label: Maintenance
+slug: /wxm-devices/pulse-bundle/maintenance
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+## WS1001 Weather Station
+
+The maintenance of the weather station is an important process, which should be conducted meticulously over the suggested period of time.
+This process includes the cleaning of certain parts of the outdoor unit and the replacement of batteries.
+The maintenance process is summarised in the following steps:
+
+<div class="explanation">
+  <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/maintenance.svg")} />
+  <div>
+    <h4>Instructions for opening the rain gauge</h4>
+    <ol>
+      <li>Clean the rain gauge at least once every 3 months.</li>
```

Comment:
Numbered list does not correspond to the number on the graphic. Is this reused somehow for D1 too? If not replicate change there too